### PR TITLE
Adds read and write local store methods

### PR DESCRIPTION
**Pull request by mko-sci**
Fixes: [SIRI-608](https://scireum.myjetbrains.com/youtrack/issue/SIRI-608)

### CODE REVIEW

File: src/main/resources/default/assets/common/core.js.pasta [hunk 1]
```diff
@@ -451,3 +451,41 @@ sirius.resetMobileZoom = function () {
...
+ * @param name the name of the local storage variable
+ * @returns the value in the local storage variable or null
+ */
+sirius.readLocalStore = function (name) {
+    try {
+        if (isDefined(window.localStorage)) {
+            return window.localStorage.getItem('sirius.' + name);
+        }
+    } catch (e) {
+        log('Cannot read local store', {name: name, error: e});
```
Comment (idlira) [inline src/main/resources/default/assets/common/core.js.pasta:468]: Do we wanna log this every time when the browser wont support it?

File: src/main/resources/default/assets/common/core.js.pasta [hunk 2]
```diff
@@ -451,3 +451,41 @@ sirius.resetMobileZoom = function () {
...
+sirius.writeLocalStore = function (name, value) {
+    if (!isDefined(value)) {
+        value = '';
+    }
+    try {
+        if (isDefined(window.localStorage)) {
+            window.localStorage.setItem('sirius.' + name, value);
+        }
+    } catch (e) {
+        log('Cannot write local store', {name: name, value: value, error: e});
```
Comment (idlira) [inline src/main/resources/default/assets/common/core.js.pasta:489]: s.a

File: src/main/resources/default/assets/common/core.js.pasta (hunk 1 above)
Comment (mko-sci) [reply src/main/resources/default/assets/common/core.js.pasta:468]: Its the same way in oxomi.readLocalStore and oxomi.writeLocalStore.
Therefore I thought thats the way to go

Comment (mko-sci) [reply src/main/resources/default/assets/common/core.js.pasta]: not necessarily :D

**Review by sabieber — APPROVED** [review]

**Review by andyHa — APPROVED** [review]

**Pull request merged**